runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/FieldTemplate/AdditionalPropertiesField.tsx

```diff
@@ -61,17 +67,24 @@ const WrapIfAdditional = ({
                 />
             </Grid>
             <Grid item xs={12}>
-                {children}
+                <ElementAwareAutocompleteWidget
+                    {...children.props.children[0].props.children[0].props}
```

Comment:
move it to const and spread this const, no idea what is it

are you sure that there is always min 1 child?